Formula/packetq.rb

```diff
@@ -0,0 +1,22 @@
+class Packetq < Formula
+  desc "SQL-frontend to PCAP-files"
+  homepage "https://www.dns-oarc.net/tools/packetq"
+  url "https://github.com/DNS-OARC/PacketQ/archive/v1.3.1.tar.gz"
+  sha256 "7c649207cc35a19fa005f09ba368466606b5d4c20715d5ef0decb21f329b568c"
+  head "https://github.com/DNS-OARC/PacketQ.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "false"
```

Comment:
We still need a test block that verifies the software works. Take a look at `fsql`'s and `govendor`'s test blocks for some examples.